src/main/scala/xiangshan/frontend/NewFtq.scala

```diff
@@ -82,10 +82,18 @@ class Ftq_RF_Components(implicit p: Parameters) extends XSBundle with BPUUtils {
         getOffset(startAddr)+offset, 0.U(instOffsetBits.W))
   }
   def getFallThrough() = {
-    getFallThroughAddr(this.startAddr, this.carry, this.pftAddr)
+    def getHigher(pc: UInt) = pc.head(VAddrBits-log2Ceil(PredictWidth)-instOffsetBits-1)
+    val startHigher = getHigher(startAddr)
+    val nextHigher  = getHigher(nextRangeAddr)
```

Comment:
should not use the higher bits of nextRangeAddr
there should be another 'nextRangeAddr' calculated as startAddr + 64